pages/audits.js

```diff
@@ -0,0 +1,73 @@
+// node_modules
+import _ from "lodash";
+import PropTypes from "prop-types";
+// component
+import AuditTable from "../components/audit-table";
+import PagePreamble from "../components/page-preamble";
+// lib
+import errorObjectToProps from "../lib/errors";
+import FetchRequest from "../lib/fetch-request";
+import { snakeCaseToHuman } from "../lib/general";
+
+export default function AuditDoc({ auditDoc }) {
+  const keyedAudits = Object.keys(auditDoc).map((key) => {
+    return {
+      key: key.split(".")[2],
+      ...auditDoc[key],
+    };
+  });
+  const auditsGroupedByCollection = _.groupBy(keyedAudits, "key");
+  return (
+    <>
+      <PagePreamble />
+      <p>
+        The IGVF Data and Administration Coordinating Center has established an
+        audit system, utilizing flags, to detect discrepancies in the data.
+        While dependencies ensure metadata accuracy within an individual object,
+        audits primarily focus on validating metadata between linked objects.
+        For example, a specific audit has been implemented to verify the linkage
+        of Biosamples with donors sharing the same taxa. Within each audit
+        category, there could be one or more icons, each assigned a distinct
+        color corresponding to the severity level of the audit category.
+      </p>
+      {Object.keys(auditsGroupedByCollection).map((itemType) => {
+        const typeAudits = auditsGroupedByCollection[itemType];
+        const filteredAudits = typeAudits.filter(
+          (audit) => audit.audit_levels[0] !== "INTERNAL_ACTION"
+        );
+        if (filteredAudits.length > 0) {
+          return (
+            <>
```

Comment:
During run time, getting a missing key error. Need to change the `<>` to `<Fragment key={itemType}>`.

Reply:
@forresttanaka` itemType` isn't defined until later in L33 so can i do `<Fragment key={keyedAudits}>`?

Reply:
@khzlin You need to change lines 40 and 45, so `itemType` exists at that point.